package/MDAnalysis/lib/distances.py

```diff
@@ -850,3 +850,49 @@ def apply_PBC(incoords, box, backend="serial"):
                backend=backend)
 
     return coords
+
+
+def calc_distance(a, b, box=None):
+    """Distance between a and b
+
+    Parameters
+    ----------
+    a, b : numpy.ndarray
+      single coordinate vectors
+    box : numpy.ndarray, optional
+      simulation box, if given periodic boundary conditions will be applied
```

Comment:
I think somewhere in the docs we will have to describe in more detail what exactly we mean and do when we say "periodic boundary conditions will be applied".